sample_data/malaysia_by_state_since_2001/malaysia_by_state_since_2001.md

```diff
@@ -0,0 +1,11 @@
+# Sources
+
+## malaysia_by_state_since_2001.geojson
+Runfola D, Anderson A, Baier H, Crittenden M, Dowker E, Fuhrig S, et al. (2020) 
+geoBoundaries: A global database of political administrative boundaries. 
+PLoS ONE 15(4): e0231866. https://doi.org/10.1371/journal.pone.0231866. 
+Downloaded from: https://github.com/wmgeolab/geoBoundaries/tree/main/releaseData/gbOpen/MYS/ADM1 on 4 July 2022.
+
+## malaysia_population_2022.csv
+Source: Department of Statistics Malaysia. 2020 Population Census. Downloaded 30 June 2022 from https://www.mycensus.gov.my/.
```

Comment:
Could you provide a more specific link?

Also, the latest population data on that website seems to be the 2020 census. Should not we change the filename's year to 2020?